.github/workflows/ci-selenium.yml

```diff
@@ -0,0 +1,25 @@
+name: Selenium Smoke Test
+
+on: [pull_request]
+
+jobs:
+  selenium:
+    runs-on: ubuntu-latest
+    container: openstax/rex-web:CI
+    steps:
+    - uses: actions/checkout@v2
+    - name: find deployment
+      env:
+        GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        . concourse/common/get_review_environment
+        review_url=$(get_review_environment)
```

Comment:
I'm not really familiar with bash. Is this `${get_review_environment}` a returned valud from `. concourse/common/get_revview_environment`?

Reply:
`get_review_environment` is a function defined in `concourse/common/get_review_environment`. the `. <filename>` sources the file so you can call the function in it.

the `jq: error (at <stdin>:4): Cannot index object with number` error happens when the github api is rate limited. i'm going to do some testing today about that